api/src/client/dynamics/DynamicsLicenseApplicationIdClient.ts

```diff
@@ -38,10 +38,11 @@ export const DynamicsLicenseApplicationIdClient = (
         const mainActiveApplications = response.data.value
           .filter(
             (applicationDetails: LicenseApplicationIdApiResponse) =>
-              applicationDetails.statecode === ACTIVE_STATECODE
+              applicationDetails.statecode !== null && applicationDetails.statecode === ACTIVE_STATECODE
           )
           .filter(
             (applicationDetails: LicenseApplicationIdApiResponse) =>
+              applicationDetails.rgb_number !== null &&
```

Comment:
[pebble] Are you able to leverage JS truthiness for both of these tests? Something along the lines of:

```typescript
applicationDetails.rgb_number && ...
```

Reply:
That's a good point, changed it!